### DISCUSSION TITLE
Block Search Engine

### DISCUSSION THREAD

**Discussion by danselstory** [AdguardTeam/AdGuardHome#3543] [Q&A]
Have a question or an idea? Please search it [on our forum](https://github.com/AdguardTeam/AdGuardHome/discussions) to make sure it was not yet asked. If you cannot find what you had in mind, please [submit it here](https://github.com/AdguardTeam/AdGuardHome/discussions/new).

### Prerequisites

Please answer the following questions for yourself before submitting an issue. **YOU MAY DELETE THE PREREQUISITES SECTION.**

- [ ] I am running the latest version
- [ ] I checked the documentation and found no answer
- [ ] I checked to make sure that this issue has not already been filed

### Additional Information

Is there a way I can block search engine on individual clients? Also like block youtube.com but not youtubekids.com?

Thank you !

**Comment by ameshkov** [comment]
The proper way would be to use the `$client` modifier: https://github.com/AdguardTeam/AdGuardHome/wiki/Hosts-Blocklists#client

You can add rules like that:
`||youtube.com^$client=ClientName`
